system-variables.md

```diff
@@ -335,6 +335,16 @@ This variable is an alias for `last_insert_id`.
 >
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
+### max_prepared_stmt_count
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Type: Integer
+- Default value: `-1`
+- Range: `[-1, 1048576]`
+- Indicates the maximum number of [`PREPARE` statement](/sql-statements/sql-statement-prepare.md) of a session.
+- `-1` means no limitation.
```

Comment:
Could you describe what happens if the the number exceeds?

Reply:
OK, a new example was added.